checks/model/hosts.js

```diff
@@ -104,10 +104,45 @@ async function checkHostMemberEntry({ fix = false } = {}) {
     }
   }
 }
+
+async function checkHostActive({ fix = false } = {}) {
+  const message = 'Host with isActive=false';
+
+  const results = await sequelize.query(
+    `SELECT *
+     FROM "Collectives"
+     WHERE "isHostAccount" IS TRUE
+     AND "deletedAt" IS NULL AND "deactivatedAt" IS NULL
+     AND "approvedAt" IS NOT NULL AND "HostCollectiveId" = "id"
+     AND "isActive" IS FALSE
+     AND COALESCE(("settings"->'forceInactive')::boolean), false) IS FALSE`,
```

Comment:
To make it a bit more explicit

```suggestion
     AND COALESCE(("settings"->'forceInactiveHost')::boolean), false) IS FALSE`,
```